### ISSUE TITLE
启动必须在虚拟环境中吗（本地环境是window）

### ISSUE THREAD

**Issue by GhnGG** [apitable/apitable#1690]
## Any questions?

**Comment by mobei12** [comment]
为了用make这个工具，windows配一个wsl,或者cgy都可以，
"We use make as our centric build tool entry that drives other build tools like gradle / npm / pnpm.
So you can just enter make command and see all build commands:" 官方原话

**Comment by lir2002** [comment]
Cgywin需要修改脚本，其中script/export-env.sh没有支持Cygwin的选项会导致环境变量加载失败。另外mysql数据文件夹mount到本地文件夹也会一直导致容器运行失败，不明白为何不创建volume来mount

**Comment by mr-kelly** [comment]
Yes, since APITable heavily relies on Unix-like build tools (like `make` and specific bash scripts), running it natively on Windows can be quite challenging. We strongly recommend using **WSL2** on Windows, as it provides a fully compatible Linux environment and works seamlessly with Docker Desktop.

---

是的，由于 APITable 高度依赖类 Unix 的构建工具（如 `make` 和 bash 脚本），在 Windows 纯本地环境运行会有很多兼容性问题。我们强烈推荐在 Windows 下使用 **WSL2**，它能提供完整的 Linux 环境，并且与 Docker Desktop 完美配合，能避免诸多文件挂载和路径报错。
